KMaina/catchup1
# Add an  about page

### What does this PR do?

It creates the about page for the site

### Description of Task to be completed?

- Create the about.html page
- Add the boilerplate code to the page
- Add h1 tags

### How should this be manually tested?
- Clone the project locally.
- `git pull` to get the new changes
- `git checkout ft-create-about-page` to go to the branch
- Open the server in your favorite browser and check for an About text in h1 tags
